docs/usage/configuration-options.md

```diff
@@ -164,7 +164,7 @@ Important: Renovate won't automerge on GitHub if a PR has a negative review outs
 <!-- prettier-ignore -->
 !!! note
     On Azure there can be a delay between a PR being set as completed by Renovate, and Azure merging the PR / finishing its tasks.
-    Renovate tries to delay until Azure is in the expected state, however if it takes too long it will continue.
+    Renovate tries to delay until Azure is in the expected state, but if it takes too long it will continue.
```

Comment:
I think this would be even clearer:
```suggestion
    Renovate tries to delay until Azure is in the expected state, but it will continue if it takes too long.
```

Reply:
That's way better! 😄